cmd/opm/validate/validate.go

```diff
@@ -0,0 +1,50 @@
+package validate
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/pkg/lib/config"
+)
+
+func NewConfigValidateCmd() *cobra.Command {
+	validate := &cobra.Command{
+		Use:   "validate <directory>",
+		Short: "Validate the declarative index config",
+		Long:  "Validate the declarative config JSON file(s) in a given directory",
+		Args: func(cmd *cobra.Command, args []string) error {
+			return cobra.ExactArgs(1)(cmd, args)
+		},
+		RunE: configValidate,
+	}
+
+	validate.Flags().BoolP("debug", "d", false, "enable debug log output")
+	return validate
+}
+
+func configValidate(cmd *cobra.Command, args []string) error {
+	debug, err := cmd.Flags().GetBool("debug")
+	if err != nil {
+		return err
+	}
+
+	logger := logrus.WithField("cmd", "validate")
+	if debug {
+		logger.Logger.SetLevel(logrus.DebugLevel)
+	}
+
+	if _, err := os.Stat(args[0]); os.IsNotExist(err) {
+		logger.Error(err.Error())
```

Comment:
I'd need to double-check but if we run into an error while stat-ing that declarative config directory, this will just pass through to that ValidateConfig function, no? Do we need to make this a `logger.Fatal(...)` or do we need to also return an error after logging that same error?

Reply:
Also, I think we should use a variable that holds the value of args[0] instead of passing args[0] to both the os.Stat and config.ValidateConfig functions.

Reply:
The load function should catch this but I think we should just return here given there is no point to continue when the directory doesn't exist.

Reply:
Another minor nit, but if we're just going to return an error, let's do the upfront checks first, like getting the `debug` flag or stat-ing the declarative config directory first, before setting up any loggers.